panda_robot_server/src/panda_robot_server/ros_bridge.py

```diff
@@ -167,6 +182,14 @@ def get_state(self):
 
     def set_state(self, state_msg):
         # Set environment state
+        if self.real_robot:  # Farhang: when using real robot. the velocity message should not suddenly stop publishing.
```

Comment:
When merged the commit will have you as author, so I would not add the your name to the comment :)

Reply:
sure thing! It was a reminder for myself to know which lines I was working on. Then just forgot to remove them.